solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -757,13 +757,17 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         (oldReserveBalances[0], oldReserveBalances[1]) = reserveBalances();
 
         uint256 amount;
+        uint256[2] memory reserveAmounts;
 
         // calculate the amount of pool tokens to mint for the caller
         // and the amount of reserve tokens to transfer from the caller
         if (totalSupply == 0) {
             amount = MathEx.geometricMean(_reserveAmounts);
+            for (uint256 i = 0; i < 2; i++) {
+                reserveAmounts[i] = _reserveAmounts[i];
```

Comment:
We don't need a for loop here

Reply:
I reverted to how it was before, but NP.

Reply:
done